blocks.md

```diff
@@ -811,44 +830,26 @@ Key | Values | Required | Default
 
 #### Available Format Keys
 
-  Key    | Value
```

Comment:
There's no way to keep these around is there? (Deprecate first and remove in future release). If not, need to figure out how to document the changes. Perhaps can re-add this table as "Removed Format Keys" and add a column showing what the new equivalent placeholder is?

Reply:
There is no sane way to keep them. I can imagine some hacky ways, but I really what to avoid them.

> Perhaps can re-add this table as "Removed Format Keys" and add a column showing what the new equivalent placeholder is?

Good idea.